admin/CMakeLists.txt

```diff
@@ -6,9 +6,11 @@ if(${BUILD_TESTS})
 endif()
 configure_file("worker_v2.m.template" "${WORKER_CONFIG_DIR}/worker_v2.m")
 
+set(_path_regex "[a-zA-Z0-9/\\\\_\\-\\.: ]+")
```

Comment:
can you add a comment explaining what this regex matches?

Reply:
Fixed in #71